applications/ParticleMechanicsApplication/custom_elements/mpm_updated_lagrangian.cpp

```diff
@@ -998,9 +998,12 @@ void MPMUpdatedLagrangian::UpdateGaussPoint( GeneralVariables & rVariables, cons
     }
 
     /* NOTE:
-    Another way to update the MP velocity (see paper Guilkey and Weiss, 2003).
-    This assume newmark (or trapezoidal, since n.gamma=0.5) rule of integration*/
-    mMP.velocity = MP_PreviousVelocity + 0.5 * delta_time * (MP_acceleration + MP_PreviousAcceleration);
+    Update velocity with Newmark integration rule 
+    This assumes newmark (since n.gamma=0.5 and beta=0.25) rule of integration*/
+    if (std::sqrt(MP_PreviousAcceleration[0]*MP_PreviousAcceleration[0]+MP_PreviousAcceleration[1]*MP_PreviousAcceleration[1]+MP_PreviousAcceleration[2]*MP_PreviousAcceleration[2])<std::numeric_limits<double>::epsilon())
```

Comment:
```suggestion
    const double norm_MP_PreviousAcceleration = std::sqrt(MP_PreviousAcceleration[0]*MP_PreviousAcceleration[0]+MP_PreviousAcceleration[1]*MP_PreviousAcceleration[1]+MP_PreviousAcceleration[2]*MP_PreviousAcceleration[2]);
    
    if (norm_MP_PreviousAcceleration < std::numeric_limits<double>::epsilon())
```

Maybe defining an additional variable the code is a bit easier to read